components/content/simple-editor/index.vue

```diff
@@ -0,0 +1,431 @@
+        
+        <button
+          class="dark:bg-gray-600"
+          @click="addSphereSceneNode"
+        >
+          Add Sphere
+        </button>
+      </div>
+      <div class="flex flex-col flex-grow gap-2">
+        <div
+          v-for="sceneNode in state.sceneNodes"
+          :key="sceneNode.id"
+          class="flex justify-between"
+          :class="{ 'ring-2 ring-blue-400': sceneNode.id === selectedNodeId }"
+        >
+          <div @click="selectSceneNode(sceneNode.id) ">
+            {{ sceneNode.type }} 
+          </div>
+          <div @click="handleDeleteSceneNode(sceneNode.id)">
+            x
+          </div>
+        </div>
+      </div>
+    </div>
+    <div
+      class="flex flex-col flex-grow"
+    >
+      <div class="flex gap-2 border border-b-0 border-gray-200 dark:border-gray-700 bg-gray-50 dark:bg-gray-500 p-1">
+        <div class="flex gap-1">
+          <button
+            v-for="mode in modes"
+            :key="mode"
+            class="dark:bg-gray-600"
+            :class="{ 'ring-2 ring-blue-400': currentMode === mode }"
+            @click="currentMode = mode"
+          >
+            {{ mode }}
+          </button>
+        </div>
+        |
+        <div class="flex gap-1">
+          <button
+            class="dark:bg-gray-600"
+            :disabled="!history.canUndo"
+            @click="history.undo"
+          >
+            Undo
+          </button>
+          <button
+            class="dark:bg-gray-600"
+            :disabled="!history.canRedo"
+            @click="history.redo"
+          >
+            Redo
+          </button>
+        </div>
+        |
+        <div class="flex gap-1">
+          <button
+            v-for="space in spaces"
+            :key="space"
+            class="dark:bg-gray-600"
+            :class="{ 'ring-2 ring-blue-400': currentSpace === space }"
+            @click="currentSpace = space"
+          >
+            {{ space }}
+          </button>
+        </div>
+        |
+        <div class="flex gap-1">
+          <button
+            v-for="cameraName in ['render', 'nav']"
+            :key="cameraName"
+            class="dark:bg-gray-600"
+            :class="{ 'ring-2 ring-blue-400': currentCameraName === cameraName }"
+            @click="currentCameraName = cameraName"
+          >
+            {{ cameraName }}
+          </button>
+        </div>
+      </div>
+      <div
+        class="flex flex-grow items-center justify-center border border-slate-200 dark:border-gray-700"
+      >
+        <ContainElement
+          :enabled="currentCameraName === 'render'"
+          :aspect-ratio="state.sceneSettings.width / state.sceneSettings.height"
+          class="border border-slate-400"
+        >
+          <div
+            ref="canvasContainerRef"
+            class="h-full w-full"
+          >
+            <TresCanvas
+              v-if=" cameraNode != null"
+              ref="tresCanvasRef"
+              clear-color="#FAFAFA"
```

Comment:
(non blocking) 
what do you think about changing the background to something less intense (like: #39B58E) I check this in the night and was too much white :)

Reply:
I'll make it gray as green made the cone color disappear (the 1st cone will have same color as tresjs logo).